Jenkinsfile

```diff
@@ -12,28 +12,34 @@ pipeline {
         }
         stage('Scan') {
             steps {
-                echo 'Scanning..'
-                sh '''
-                      FASTAPI_ENV=test python3 -m uvicorn src.main:app &
-                      curl -Lo mapi https://mayhem4api.forallsecure.com/downloads/cli/latest/linux-musl/mapi && chmod +x mapi
-                   '''
-                withCredentials([string(credentialsId: 'MAPI_TOKEN', variable: 'MAPI_TOKEN')]) {
-                    sh './mapi login ${MAPI_TOKEN}'
+                catchError(buildResult: 'FAILURE', stageResult: 'FAILURE') {
```

Comment:
Hmm, why this catchError?

Reply:
@unionfindbee ☝️

Reply:
I was trying to make sure the coverage report would show even if the build failed. 

In Jenkins with Cobertura, the coverage report doesn't natively display when the build fails

Reply:
This would argue more for a zero exit code for runs where MAPI finds anything and failing the pipeline due to errors in the SARIF report - not sure if Jenkins supports that. For a later date!